client/client.go

```diff
@@ -86,7 +86,7 @@ func New(cfg *config.Config) (*DatabricksClient, error) {
 
 type DatabricksClient struct {
 	Config *config.Config
-	client *httpclient.ApiClient
+	Client *httpclient.ApiClient
```

Comment:
Does exporting this come with unforeseen risks? We may want to include the rationale for exporting/unexporting this in a field comment (there must have been a reason to unexport it to begin with).

Reply:
I was trying to figure out a convenient way to handle this without exporting the client. Let me take another stab. The thing is that there are some complex dependencies, as the API client doesn't depend on the `config` package, but these convenience constructors do. `config` doesn't depend on `client`, which is why these convenience constructors can live here.